examples/triply/vertexBufferNode.cpp

```diff
@@ -82,14 +82,24 @@ void VertexBufferNode::setupTree(VertexData& data, const Index start,
         ++progress;
 }
 
-/*  Compute the bounding sphere from the children's bounding spheres.  */
-const BoundingSphere& VertexBufferNode::updateBoundingSphere()
+void VertexBufferNode::updateBounds()
 {
-    // take the bounding spheres returned by the children
-    const BoundingSphere& sphere1 = _left->updateBoundingSphere();
-    const BoundingSphere& sphere2 = _right->updateBoundingSphere();
+    _left->updateBounds();
+    _right->updateBounds();
+
+    // compute enclosing box
+    const auto& box1 = _left->getBoundingBox();
+    const auto& box2 = _right->getBoundingBox();
+    _boundingBox[0][0] = std::min(box1[0][0], box2[0][0]);
```

Comment:
Not an AABB, but a Vector3f[2]. Might clean it up with another improvement in progress.